### PR TITLE
[14.0][MIG] delivery_carrier_partner

### PR DESCRIPTION
**Pull request by ValentinVinagre**
Standard Migration

### CODE REVIEW

**Comment by ValentinVinagre** [comment]
@HaraldPanten

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
@pedrobaeza @JordiBForgeFlow do you mind fast-tracking this simple one, please?

Thx!

File: delivery_carrier_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery carrier partner",
+    "summary": "Add a partner in the delivery carrier",
+    "version": "14.0.1.0.1",
```
Comment (pedrobaeza) [inline delivery_carrier_partner/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../../delivery_carrier_partner
```
Comment (pedrobaeza) [inline setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner:1]: This should go in a previous commit, as it's automatic.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner (hunk 2 above)
Comment (ValentinVinagre) [reply setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner:1]: I delete this part?

Comment (pedrobaeza) [reply setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner:1]: OK, it's not too important. Let's keep it this way.

**Review by pedrobaeza — APPROVED** [review]

**Comment by ValentinVinagre** [comment]
> /ocabot merge nobump

Thanks @pedrobaeza

**Pull request merged**